src/layout/components/Navbar.vue

```diff
@@ -32,59 +38,111 @@
 </template>
 
 <script>
-import { mapGetters } from 'vuex'
-import Breadcrumb from '@/components/Breadcrumb'
-import Hamburger from '@/components/Hamburger'
+import LayoutHamburger from './Hamburger'
+
+function importSlotsFromLayout() {
+  const slots = this.layout.$slots
+  if ('avatar' in slots) {
+    this.$slots.avatar = slots.avatar
+  }
+  if ('avatarDropdown' in slots) {
+    this.$slots.avatarDropdown = slots.avatarDropdown
+  }
+  if ('navbarLeft' in slots) {
+    this.$slots.navbarLeft = slots.navbarLeft
```

Comment:
*Use-Case* on the top level `<Layout />` component, I want to be able to replace that slot.

e.g.

```html
<layout>
  <template #navbarLeft>
    <!-- Add more items, defined from usage time from the top level component, here. -->
    <div class="navbar-left-item">
      <breadcrumb />
    </div>
  </template>
</layout>
```

Reply:
Feels related to [this article **Advanced Vue.js Controlling Parent Slits**](https://michaelnthiessen.com/advanced-vue-controlling-parent-slots)